app/assets/javascripts/answers.js

```diff
@@ -9,11 +9,16 @@ function sendFeedback(feedbackUrl, isCorrect) {
 function handleUserFeedback(feedbackActive, feedbackString, feedbackQid) {
   if (feedbackActive) { sendFeedback('/questions/' + feedbackQid + '/feedback', feedbackString);}
   if (feedbackString == 'yes') {
-    $('.feedback-alert').removeClass('alert-danger').addClass('alert-success');
-    $('.feedback-alert-text').text('Great! Congrats! Try another question.');
-  } else {
-    $('.feedback-alert').removeClass('alert-success').addClass('alert-danger');
-    $('.feedback-alert-text').text("No worries. You'll get it next time. Onwards!");
+    $('.feedback-alert').removeClass('alert-danger').removeClass('alert-maybe').addClass('alert-success');
```

Comment:
This can be simplified to `.removeClass('alert-danger alert-maybe')`, a little shorter. You can do the same for the others below.